data/manual/FAQ.txt

```diff
@@ -70,7 +70,7 @@ Zim is written as a "single user" program, so it is not intended for multiple pe
 ===== How do I change the Gtk theme? =====
 On a Linux desktop, you can probably do this via your settings manager.
 
-Windows users and users please see the section "Gtk configuration" in [[Help:Config Files]]
+Windows users and users please see the section "Gtk configuration" in [[Help:Config Files|Config Files]]
```

Comment:
Should this say "Windows and $mac_os users"? (no idea what MacOSX is called nowadays)

Reply:
Most probably yes (the manual refers to Mac as _macOS_), although I'm not familiar with macOS. The text was added in a66477e136e7, so in case of doubt we should ask @jaap-karssenberg .

Reply:
Yes, agree it should be windows and Mac users